pkg/tinydtls/contrib/sock_dtls.c

```diff
@@ -813,7 +813,13 @@ static void _ep_to_session(const sock_udp_ep_t *ep, session_t *session)
     session->port = ep->port;
     session->size = sizeof(ipv6_addr_t) +       /* addr */
                     sizeof(unsigned short);     /* port */
-    session->ifindex = ep->netif;
+    if ((ep->addr.ipv6[0] == 0xfe) && (ep->addr.ipv6[0] == 0x80)) {
```

Comment:
```suggestion
    if ((ep->addr.ipv6[0] == 0xfe) && (ep->addr.ipv6[1] == 0x80)) {
```

Reply:
I wanted to prevent a dependency to the `ipv6_addr` module. But you are right, that one is an inline function ^^"

Reply:
Done and squashed.